charts/camunda-platform/templates/NOTES.txt

```diff
@@ -66,6 +74,14 @@ If you don't have an Ingress Controller you can use kubectl port-forward to acce
 
 > kubectl port-forward svc/{{ .Release.Name }}-tasklist 8081:80
 
+    {{- if .Values.global.tasklist.auth.identity.enabled }}
+
+Per default authentication via Identity/Keycloak is enabled. In order to login into Tasklist please port-forward to Keycloak
```

Comment:
:wrench: Same as the other PR, wording is kind of funky here, I would remove the `Per default` and only mention instead to change it if they didn't want it.